ignitionrobotics/ign-gazebo
# Added lidar component inspector

Signed-off-by: Nate Koenig <nate@openrobotics.org>

# 🎉 New feature

## Summary

Added a lidar component inspector.

## Test it

1. `ign gazebo -v 4 sensors.sdf`
2. Take note of the output from `ign model -m sensors_box -l link -s laser`
3. Select the laser sensor in the entity tree.
4. Make some changes to the values in the component inspector. 
5. Rerun `ign model -m sensors_box -l link -s laser` and you should see the changes.

## Checklist

**Note to maintainers**: Remember to use **Squash-Merge**
